lib/Handler/FileHandler.php

```diff
@@ -71,8 +70,12 @@ public function write($level, $message, array $splitFullMessage, array $context
      */
     public function writeToFile($line)
     {
-        if (fwrite($this->handle, $line) === false) {
-            throw new \Exception('Cannot write to file: '.$this->handle);
+        $res = file_put_contents($this->filename, $line, FILE_APPEND | LOCK_EX);
+
+        if ($res === false) {
+            throw new HandlerException('Cannot write to file: ' . $this->filename);
```

Comment:
Not needed, returning `false` is enough. The exception should be thrown a level higher (as it's already done).